test/functional/test-insert-extension.js

```diff
@@ -178,7 +178,11 @@ describe('test-insert-extension', () => {
       expect(getMode().localDev).to.be.true;
       const script = calculateExtensionScriptUrl('examples.build/ads.amp.html',
           'amp-ad', true);
-      expect(script).to.equal('/base/dist/v0/amp-ad-0.1.js');
+      if (window.ampTestRuntimeConfig.useCompiledJs) {
```

Comment:
I feel in a test we should expect only one result. Is there a way to set `window.ampTestRuntimeConfig.useCompiledJs` and then create two tests to check this condition?